### ISSUE TITLE
Distribution of ATLAS objects

### ISSUE THREAD

**Issue by MatthewJA** [chengsoonong/crowdastro#92]
@jbanfield: How are the ATLAS objects distributed through the database? Are they roughly uniform in type or should I expect e.g. more compact sources early on? (When sorting by Zooniverse ID.)

Also, I can compile a list of objects that volunteers and my classifier find hard to classify (in particular, when _both_ are hard to classify). While I'm at it, are there any other lists of objects you'd like?

**Comment by MatthewJA** [comment]
Also also &mdash; are there any expert classifications for ATLAS? The gold standard is only for FIRST as far as I can tell. I note that Norris et al. (2006) did make a catalogue but they estimate that 9% of their classifications are incorrect (though that's not too different to the estimate in your paper).

**Comment by MatthewJA** [comment]
Mostly for my own reference, the cross-identifications of Norris et al. are [here](http://vizier.cfa.harvard.edu/viz-bin/VizieR-3?-source=J/AJ/132/2409/table6).

**Comment by jbanfield** [comment]
There is no gold standard for ATLAS.  Can the gold standard be used for ATLAS too?  Not sure.  Maybe just the expert classifications from Norris et al.

The ATLAS sources are more compact then anything else.  We do not have other information.  They were sorted by radio flux density.  I do not know if they were the Zooniverse ids reflect this.

**Comment by MatthewJA** [comment]
Thanks! I'll try and figure out how to use the Norris et al. classifications.

**Comment by MatthewJA** [comment]
Got it, I think! There are ~700 Norris et al. classifications for ATLAS&ndash;CDFS, of which ~300 don't appear in our data at all. Discarding subjects that have no corresponding Norris classification (n = 167):

LR agreement with RGZ: 77.25%
LR agreement with Norris: 74.85%
LR agreement with both: 63.47%
LR agreement with either: 88.62%

RF agreement with RGZ: (0.89 +- 0.033)%
RF agreement with Norris: (0.54 +- 0.033)%
RF agreement with both: (0.51 +- 0.026)%
RF agreement with either: (0.93 +- 0.027)%

Here, uncertainties are standard deviations over 10 independently trained random forest classifiers.

I can also compute how confident the classifier or RGZ are for each data point, but I'm not sure how to present that.

**Comment by chengsoonong** [comment]
1. Please write the above experimental protocol and result in a LaTeX document (thesis chapter).
2. Present the distribution of confidences (of all data points) as a violin plot.

**Comment by MatthewJA** [comment]
Can do!

FWIW I think that throwing out subjects with no Norris classification is a bit dodgy &mdash; not sure how to deal with that.

**Comment by jbanfield** [comment]
I guess the question is "what are the radio subjects with no Norris classification?"  I would guess that these are the fainter flux density radio subjects as the Norris full image has a higher noise level then the one from Franzen+(2015).

**Comment by chengsoonong** [comment]
Two separate "throw out":
1. training set
2. test set.

Test set: My advice is report performance for all possible cells in the Venn diagram. The reader can then take whichever cell he/she is excited about.

**Comment by MatthewJA** [comment]
Carried into #105.
